package.json

```diff
@@ -29,8 +29,8 @@
     "@storybook/storybook-deployer": "^2.0.0",
     "@testing-library/jest-dom": "^5.3.0",
     "@testing-library/react": "^10.0.2",
-    "@theforeman/builder": "^4.3.0",
-    "@theforeman/vendor-dev": "^4.3.0",
+    "@theforeman/builder": "^4.2.0",
+    "@theforeman/vendor-dev": "^4.2.0",
```

Comment:
Curious why the downgrade?

Reply:
I gotta change it back, bad git rebase! Really this should be 4.5.0 like https://github.com/theforeman/foreman/pull/7519 - good catch